src/main/java/controller/RaceGame.java

```diff
@@ -0,0 +1,45 @@
+package controller;
+
+import objects.Car;
+import objects.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class RaceGame {
+    int gameCount;
+
+    private static Cars carRepository;
+
+    private static List<Car> winningCars = new ArrayList<>();
```

Comment:
```suggestion
    private Cars carRepository;

    private List<Car> winningCars = new ArrayList<>();
```
위의 코드와는 무엇이 다른지 설명해주세요!

또한, 만약 서로 다른 RaceGame이 두개 이상 필요하다면 어떻게 될까요?

Reply:
java의 클래스는 한 번만 메모리로 로드됩니다.
static으로 선언하게 되면 static 영역에 저장되어 클래스가 끝날 때 까지 계속 존재하게 됩니다. 
static이 아닌 변수를 new 로 객체를 생성하게 되면 heap영역에 저장되어 가비지 컬렉터에 의해 자동으로 관리되어 집니다.
이러한 특징으로 객채를 생성하지 않고 여러 객체가 해당 변수나 함수에 접근이 가능해집니다. 
ex) Math.random() 
 
 
 두 번째 질문 만약 서로 다른 RaceGame이 두개 이상 필요하다면 어떻게 될까요?
답변 : 고민해본 결과 서로 다른 RaceGame일 경우에는 static을 선언 하는쪽이 메모리 사용에 효과적일거라 생각합니다.  여러개의 RaceGame객채를 생성하게 되면 static으로 선언하지 않은 모든 변수들 또한 메모리에 상주하게 됩니다.
이러면 메모리 공간적으로 많이 비효율적이라 생각합니다.